wraps/python/src/indicators/SMA.py

```diff
@@ -0,0 +1,37 @@
+from Skender.Stock.Indicators import Indicator
+from cstypes import List, to_pyfloat
+from .Qoute import Qoute
+
+def get_sma(history, lookbackPeriod: int):
+    sma_list = Indicator.GetSma[Qoute](List(Qoute, history), lookbackPeriod)
+    sma_list = [ SmaResult(r) for r in sma_list ]
+
+    return sma_list
+
+def get_sma_extended(history, lookbackPeriod: int):
+    sma_extended_list = Indicator.GetSmaExtended[Qoute](List(Qoute, history), lookbackPeriod)
+    sma_extended_list = [ SmaExtendedResult(r) for r in sma_extended_list ]
+
+    return sma_extended_list
+
+class SmaResult:
+    def __init__(self,sma_result):
+        self.Date = sma_result.Date
+        self.Sma = to_pyfloat(sma_result.Sma)
+
+    def __str__(self):
```

Comment:
How is this string concatenation used?  Seems like an odd thing, but I'm not a Python expert.

Reply:
`__str__()` is used for type casting in to string. Just for convenience.
Usually used for:
```python
>>> sma_list = Indicator.GetSma[Qoute]( some values... )
>>> print(sma_list[0])
2021-09-23    12.3
```
Without `__str__()`:
```python
>>> sma_list = Indicator.GetSma[Qoute]( some values... )
>>> print(sma_list[0])
<indicators.SMA.SmaResult object at 0x0A710700>
```

Reply:
Is that used for debugging or is it needed for the production code?  The reason I ask is if it's just debugging code, it probably shouldn't be in the final production code and may not be as efficient in higher-performing use cases.  What's your opinion?

Reply:
Ah..then, It's more like debugging code. `__str__()` is only needed for string expression of the object. In my opinion, removing from production code is better.